influxql/ast.go

```diff
@@ -1448,9 +1448,6 @@ func (s *ShowSeriesStatement) RequiredPrivileges() ExecutionPrivileges {
 
 // DropSeriesStatement represents a command for removing a series from the database.
 type DropSeriesStatement struct {
-	// The Id of the series being dropped (optional)
-	SeriesID uint64
-
 	// Data source that fields are extracted from (optional)
 	Source Source
```

Comment:
I didn't update this with the regex changes a while back but it probably should be changed to `Sources Sources` and change `parseDropSeriesStatement` to call `parseSources`.  Or, we should add an argument to `parseSource` to disallow regexp.